tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/GitHubService.cs

```diff
@@ -140,28 +142,40 @@ public async Task<List<string>> CreatePullRequest(string repoName, string repoOw
                 return responseList;
             }
 
-
             // Create the pull request
-            responseList.Add($"Changes are mergeable. Proceeding to create pull request for changes in {headBranch}.");
-            var pullRequest = new NewPullRequest(title, headBranch, baseBranch)
-            {
-                Body = body
-            };
-
+            PullRequest? createdPullRequest = null;
             try
             {
-                var createdPullRequest = await gitHubClient.PullRequest.Create(repoOwner, repoName, pullRequest);
+                responseList.Add($"Changes are mergeable. Proceeding to create pull request for changes in {headBranch}.");
+                var pullRequest = new NewPullRequest(title, headBranch, baseBranch)
+                {
+                    Body = body
+                };
+
+                createdPullRequest = await gitHubClient.PullRequest.Create(repoOwner, repoName, pullRequest);
                 if (createdPullRequest == null)
+                {
                     responseList.Add($"Failed to create pull request for changes in {headBranch}.");
-                else
-                    responseList.Add($"Pull request created successfully. Pull request URL: {createdPullRequest.HtmlUrl}");
-                return responseList;
+                    return responseList;
+                }
+                responseList.Add($"Pull request created successfully. Pull request URL: {createdPullRequest.HtmlUrl}");
             }
             catch (Exception ex)
             {
                 responseList.Add($"Failed to create pull request. Error: {ex.Message}");
                 return responseList;
             }
+
+            try
+            {
+                // Add label
+                await gitHubClient.Issue.Labels.AddToIssue(repoOwner, repoName, createdPullRequest.Number, [CreatedByCopilotLabel]);
```

Comment:
Do we have to manually add the label in all the repositories for this call to work? or should there be a check first and if not present create it?